### DISCUSSION TITLE
the orange icon on dock is blank, does anyone know how to fix it? I installed orange3 using conda

### DISCUSSION THREAD

**Discussion by DylanZDD** [biolab/orange3#6122] [Q&A]
<img width="144" alt="Screen Shot 2022-09-04 at 12 26 04" src="https://user-images.githubusercontent.com/44270787/188297386-c463907c-9e7f-45ea-b46f-b0ad9b6f8f23.png">
<img width="1431" alt="Screen Shot 2022-09-04 at 12 26 13" src="https://user-images.githubusercontent.com/44270787/188297398-7584db2e-be45-4b6b-839a-f20de5185e50.png">

**Comment by markotoplak** [comment]
Thanks. I confirm what you are seeing. I never thought of needing an icon in my developer install, but it would indeed make sense. Thus we opened issue #6129.
